content/influxdb/v1.8/concepts/schema_and_data_layout.md

```diff
@@ -1,140 +1,169 @@
-  </tr>
-</table>
+  - [Where to store data (tag or field)](#where-to-store-data-tag-or-field)
+  - [Avoid too many series](#avoid-too-many-series)
+  - [Use recommended naming conventions](#use-recommended-naming-conventions)
+  - [Shard Group Duration Management](#shard-group-duration-management)
 
-## General recommendations
+## Where to store data (tag or field)
 
-### Encouraged schema design
+Your queries should guide what data you store in [tags](/influxdb/v1.8/concepts/glossary/#tag) and what you store in [fields](/influxdb/v1.8/concepts/glossary/#field) :
 
-We recommend that you:
+- Store commonly-queried and grouping ([`group()`](/flux/v0.x/stdlib/universe/group) or [`GROUP BY`](/influxdb/v1.8/query_language/explore-data/#group-by-tags)) metadata in tags.
+- Store data in fields if each data point contains a different value.
+- Store numeric values as fields ([tag values](/influxdb/v1.8/concepts/glossary/#tag-value) only support string values).
 
-- [Encode meta data in tags](#encode-meta-data-in-tags)
-- [Avoid using keywords as tag or field names](#avoid-using-keywords-as-tag-or-field-names)
+## Avoid too many series
 
-#### Encode meta data in tags
+IndexDB indexes the following data elements to speed up reads:
 
-[Tags](/influxdb/v1.8/concepts/glossary/#tag) are indexed and [fields](/influxdb/v1.8/concepts/glossary/#field) are not indexed.
-This means that queries on tags are more performant than those on fields.
+- [measurement](/influxdb/v1.8/concepts/glossary/#measurement)
+- [tags](/influxdb/v1.8/concepts/glossary/#tag)
 
-In general, your queries should guide what gets stored as a tag and what gets stored as a field:
+[Tag values](/influxdb/v1.8/concepts/glossary/#tag-value) are indexed and [field values](/influxdb/v1.8/concepts/glossary/#field-value) are not.
+This means that querying by tags is more performant than querying by fields.
+However, when too many indexes are created, both writes and reads may start to slow down.
 
-- Store commonly-queried meta data in tags
-- Store data in tags if you plan to use them with the InfluxQL `GROUP BY` clause
-- Store data in fields if you plan to use them with an [InfluxQL](/influxdb/v1.8/query_language/functions/) function
-- Store numeric values as fields ([tag values](/influxdb/v1.8/concepts/glossary/#tag-value) only support string values)
+Each unique set of indexed data elements forms a [series key](/influxdb/v1.8/concepts/glossary/#series-key).
+[Tags](/influxdb/v1.8/concepts/glossary/#tag) containing highly variable information like unique IDs, hashes, and random strings lead to a large number of [series](/influxdb/v1.8/concepts/glossary/#series), also known as high [series cardinality](/influxdb/v1.8/concepts/glossary/#series-cardinality).
+High series cardinality is a primary driver of high memory usage for many database workloads.
+Therefore, to reduce memory overhead, consider storing high-cardinality values in fields rather than in tags.
```

Comment:
```suggestion
Therefore, to reduce memory consumption, consider storing high-cardinality values in field values rather than in tags or field keys.
```